app/views/alchemy/pages/_meta_data.html.erb

```diff
@@ -0,0 +1,13 @@
+<% if @page %>
+  <%= tag(:meta, charset: "utf-8") %>
```

Comment:
Could this not be just `<meta charset="utf8">`? I don't see the need for using the `tag` helper here.